company/okrs/2020_q1.md

```diff
@@ -80,5 +80,9 @@
    1. Code intelligence KR: Implement search within dependencies and/or dependents of current repo
    1. Code intelligence KR: Implement searching for defs/refs by symbol name
 
+1. **Web: Make users aware of advanced search features**
+   1. Web KR: Shorten time to first search
```

Comment:
@lguychard do we already have a metric tracking this? If not, then the KR should be to create a metric to start tracking this. Same question for the next KR.

Reply:
Also, anything automation related?

Reply:
@nicksnyder edited to add KR about creating metrics.